chain/client/src/chunk_producer.rs

```diff
@@ -301,10 +300,9 @@ impl ChunkProducer {
                 outgoing_receipts_root,
                 tx_root,
                 congestion_info,
-                chunk_extra.bandwidth_requests().cloned(),
+                chunk_extra.bandwidth_requests().cloned().unwrap_or_default(),
```

Comment:
not sure if this is better or trying to change:
`pub fn bandwidth_requests(&self) -> Option<&BandwidthRequests>`
to

`pub fn bandwidth_requests(&self) -> &BandwidthRequests`
possibly by panic on matching V1 to V3 or removing these as well.

Reply:
Since these cannot be removed I think retaining the Option & dealing with None at the callsite is better than a panic.